wadas/ui/mainwindow.py

```diff
@@ -454,16 +454,24 @@ def update_info_widget(self):
 
         if OperationMode.cur_operation_mode_type:
             self.ui.label_op_mode.setText(OperationMode.cur_operation_mode_type.value)
+            match OperationMode.cur_operation_mode_type:
+                case OperationMode.OperationModeTypes.TunnelMode:
+                    classification_en_txt = "No"
+                case OperationMode.OperationModeTypes.AnimalDetectionMode:
+                    classification_en_txt = "No"
+                case OperationMode.OperationModeTypes.CustomSpeciesClassificationMode:
+                    classification_en_txt = "Yes (custom)"
+                case _:
+                    classification_en_txt = "Yes"
+            self.ui.label_classification_enablement.setText(classification_en_txt)
         else:
             self.ui.label_op_mode.setText("None")
+            self.ui.label_classification_enablement.setText("")
 
         if OperationMode.cur_operation_mode:
-            self.ui.label_last_detection.setText(
-                os.path.basename(OperationMode.cur_operation_mode.last_detection)
-            )
-            self.ui.label_last_classification.setText(
-                os.path.basename(OperationMode.cur_operation_mode.last_classification)
-            )
+            filename = os.path.basename(OperationMode.cur_operation_mode.last_detection)
+            last_dir = os.path.basename(os.path.dirname(OperationMode.cur_operation_mode.last_detection))
+            self.ui.label_last_detection.setText(f"{last_dir}/{filename}")
```

Comment:
It's better to use os.path.join() instead of directly using / as path separator.